test/functions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+require '../functions.php';
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+    public function testSuccessBottlesHtml()
+    {
+        $testinput = [
+            ['id' => 1, 'itemname' => 'Park Rye', 'type' => 'Rye', 'purchaselocation' => 'Canada', 'purchasedate' => '2022-01-04']
+        ];
+        $expectedOutput = "<div class='bottleCard'><h2>Park Rye</h2><p>Type: Rye</p><p>Purchase location: Canada</p><p>Date purchased: 2022-01-04</p></div>";
+        $actualOutput = bottlesHtml($testinput);
+        $this->assertEquals($expectedOutput, $actualOutput);
+    }
+
+    public function testFailuresBottlesHtml()
```

Comment:
the name of the function should probably be `testFailureBottlesHtml()` (or testFailureSomethingElse if you change the name of the other function as per my other comment)